executor/tests/storage_integration_test.rs

```diff
@@ -599,7 +599,8 @@ fn verify_transactions(
     let txns = txn_list_with_proof
         .transaction_and_infos
         .iter()
-        .map(|(txn, _)| Transaction::UserTransaction(txn.clone()))
+        .map(|(txn, _)| txn)
```

Comment:
not related to this change, but this probably can use `unzip`.

Reply:
in that world the second element needs to be cloned as well (unnecessarily), like:
`let (txns, _): (Vec<_>, Vec<_>) = txn_list_with_proof.transaction_and_infos.iter().cloned().unzip()`

I don't think it's better..